.dockerignore

```diff
@@ -1,7 +0,0 @@
-*
```

Comment:
Could you remind me why did we have to remove this `.dockerignore`?

Reply:
In order to copy src files into the container so we can build it (instead of pulling from a URL).

Reply:
Ok, got it